src/core_plugins/kibana/public/management/sections/objects/lib/import_file.js

```diff
@@ -0,0 +1,13 @@
+export async function importFile(file, FileReader = window.FileReader) {
+  return new Promise((resolve, reject) => {
```

Comment:
Should we check the `file` metadata to ensure that we _want_ to open it (assuming it's a `File` and not a `Blob`)? Perhaps its size to limit it too?